components/hook/hook.js

```diff
@@ -0,0 +1,43 @@
+import { Component } from "react";
+
+import Code from "../code/code";
+import Clipboard from "../clipboard/clipboard";
+
+class Hook extends Component {
+  constructor(props){
+    super(props);
+    this.hook = props.hook;
```

Comment:
As I mentioned above we won't be needing class component. But if we were using it then I would have asked you these questions-

Why are we doing `this.hook = props.hook`. We can remove this line and in the render function directly do

```js
const hook = this.props.hook;
```
and then simply do `hook.name`.